sdk/identity/azure-identity/azure/identity/_credentials/azure_cli.py

```diff
@@ -73,8 +74,11 @@ def get_token(self, *scopes, **kwargs): # pylint: disable=no-self-use
 
         resource = _scopes_to_resource(*scopes)
         command = COMMAND_LINE.format(resource)
-        tenant = resolve_tenant(default_tenant=self.tenant_id, **kwargs)
-
+        tenant = resolve_tenant(
```

Comment:
I think I'm a bit confused by this behavior. Because the default tenant for this credential isn't defined (`""`), any `tenant_id` passed to `get_token` will be returned here. That makes sense for the first time a tenant is provided -- we can consider that to be the tenant configured for this credential.

But if a new tenant ID is provided to `get_token` at a later time, without that new tenant being added to `additionally_allowed_tenants`, the tenant would still resolve to the new tenant. Isn't that an issue?

Reply:
Took offline